app/handler/commands.js

```diff
@@ -803,6 +804,31 @@ exports.handleMessage = function (steamID, message) {
             }
             client.chatMessage(steamID, 'Price check has been requested, the item will be checked.');
         });
+    } else if (isAdmin && command === 'name') {
+        // This has already been used but since I'm planning on rewriting this for user extensions it will remain.
+        const newName = message.substr(command.length + 1).trim();
+
+        community.editProfile({
+            name: newName
+        }, function (err) {
+            if (err) {
+                client.chatMessage(steamID, 'Error while changing bot\'s name: ' + err.message);
```

Comment:
Remove "bot's" from the messages, we already know that we are trying to change the bot's name.